proto/dydxprotocol/vault/params.proto

```diff
@@ -38,3 +38,9 @@ message Params {
     (gogoproto.nullable) = false
   ];
 }
+
+// VaultParams stores individual parameters of a vault.
+message VaultParams {
+  // The quoting parameters specific to this vault.
+  QuotingParams quoting_params = 1 [ (gogoproto.nullable) = false ];
```

Comment:
nit: This isn't nullable, how do we know we should use the default? Or is the plan to set it to equal to the default at the start?

Reply:
i was thinking about setting this to the default when a vault is created (i.e. first deposited into)

Reply:
thought more about this and simply removed `VaultParams` proto